### PR TITLE
CC-32057 Updated Marketplace Merchant, Warehouse User Management and Glue Authentication features.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Epic: https://spryker.atlassian.net/browse/CC-32057

Ticket: https://spryker.atlassian.net/browse/CC-32072

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md [hunk 1]
```diff
@@ -0,0 +1,1653 @@
...
+| MODULE                    | EXPECTED DIRECTORY                          |
+|---------------------------|---------------------------------------------|
+| MerchantProfile           | vendor/spryker/merchant-profile             |
+| MerchantProfileDataImport | vendor/spryker/merchant-profile-data-import |
+| MerchantProfileGui        | vendor/spryker/merchant-profile-gui         |
+| MerchantSearch            | vendor/spryker/merchant-search              |
+| MerchantSearchExtension   | vendor/spryker/merchant-search-extension    |
+| MerchantUser              | vendor/spryker/merchant-user                |
+| MerchantUserGui           | 	vendor/spryker/merchant-user-gui           |
+| MerchantStorage           | vendor/spryker/merchant-storage             |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md:39]: * We have to add the module to the list (OauthMerchantUser).

File: _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md [hunk 2]
```diff
@@ -0,0 +1,1653 @@
...
+console translator:generate-cache
+```
+
+### 5) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                               | DESCRIPTION                                                                                              | PREREQUISITES | NAMESPACE                                                            |
+|------------------------------------------------------|----------------------------------------------------------------------------------------------------------|---------------|----------------------------------------------------------------------|
+| MerchantProfileExpanderPlugin                        | Expands merchant with profile data.                                                                      |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md:180]: * Deprecated one, has to be substituted by the \Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant\MerchantProfileMerchantBulkExpanderPlugin.

File: _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md [hunk 3]
```diff
@@ -0,0 +1,1653 @@
...
+| MerchantProfileMerchantPostCreatePlugin              | Creates merchant profile on merchant create action.                                                      |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
+| MerchantProfileMerchantPostUpdatePlugin              | Updates merchant profile on merchant update action.m                                                     |               | Spryker\Zed\MerchantProfile\Communication\Plugin\Merchant            |
+| MerchantProfileContactPersonFormTabExpanderPlugin    | Adds an extra tab to merchant edit and create forms for editing and creating contact person data.        |               | Spryker\Zed\MerchantProfileGui\Communication\Plugin\MerchantGui\Tabs |
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md:183]: Missed plugins:
* MerchantProfileAclEntityConfigurationExpanderPlugin

Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202403.0/marketplace/install-the-marketplace-merchant-feature.md:183]: @abitskil, this plugin is in `install-the-marketplace-merchant-portal-core-feature`.

**Review by abitskil — APPROVED** [review]

File: docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-the-authentication.md [hunk 4]
```diff
@@ -65,69 +60,75 @@ vendor/bin/console transfer:generate
...
+| UserPasswordOauthRequestGrantTypeConfigurationProviderPlugin  | Builds `OauthGrantTypeConfigurationTransfer` from the configuration of Password GrantType data.          | Spryker\\Zed\\Oauth\\Communication\\Plugin\\Oauth                                      |
+| UserRequestValidatorPlugin                                    | Validates if `GlueRequestTransfer.requestUser` is set if the request has the Authorisation header.   | Spryker\\Glue\\OauthBackendApi\\Plugin\\GlueApplication                                |
+| BackofficeUserOauthScopeAuthorizationCheckerPlugin            | Executes the authorization check based on the Back Office user OAuth scope.                                   | Spryker\Zed\OauthUserConnector\Communication\Plugin\OauthUserConnector                 |
+| OauthUserScopeProtectedRouteAuthorizationConfigProviderPlugin | Provides Oauth users authorization strategy configuration.                                            | Spryker\Glue\OauthUserConnector\Plugin\GlueBackendApiApplicationAuthorizationConnector |
```
Comment (andriitserkovnyi) [inline docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-the-authentication.md:130]:
```suggestion
| OauthUserScopeProtectedRouteAuthorizationConfigProviderPlugin | Provides the configuration of Oauth user authorization strategy.                                            | Spryker\Glue\OauthUserConnector\Plugin\GlueBackendApiApplicationAuthorizationConnector |
```

File: docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-the-authentication.md [hunk 5]
```diff
@@ -65,69 +60,75 @@ vendor/bin/console transfer:generate
...
+| OauthUserScopeAuthorizationStrategyPlugin                     | Processes Oauth users an authorization request.                                                         | Spryker\Zed\OauthUserConnector\Communication\Plugin\Authorization                      |
```
Comment (andriitserkovnyi) [inline docs/scos/dev/migration-concepts/migrate-to-decoupled-glue-infrastructure/decoupled-glue-infrastructure-integrate-the-authentication.md:131]:
```suggestion
| OauthUserScopeAuthorizationStrategyPlugin                     | Processes the authorization requests of Oauth users.                                                         | Spryker\Zed\OauthUserConnector\Communication\Plugin\Authorization                      |
```

**Review by andriispryker — APPROVED** [review]

**Pull request merged**